src/components/CTypePresentation/CTypePresentation.tsx

```diff
@@ -91,7 +91,7 @@ class CTypePresentation extends React.Component<Props, State> {
               size={size || DEFAULT_SIZE}
               theme="polkadot"
             />
-            {this.getLabel(cType.cType.metadata.title.default)}
+            {this.getLabel(cType.cType.schema.$id)}
```

Comment:
A quick fix, overall I would need to change the state to match the new ctype. Unfortunately, it is a lot of work and seems out of the scope of the ticket.

Reply:
So, since we now have `CTypeWithMetadata` as a type, we can access the metadata again, which is done as of right now